packages/ui/src/mocks/data/seedWorkers.ts

```diff
@@ -37,11 +37,11 @@ export const seedWorker = (worker: RawWorker, server: any) => {
 const seedWorkerStatus = (status: WorkerStatus, id: string, server: any) => {
   switch (status) {
     case 'active':
-      return server.schema.create('WorkerStatusActive', {})
+      return server.schema.create('WorkerStatusActive', { phantom: 0 })
```

Comment:
I think I can use this in my new commit instead of my resolvers, I should just filter by phantom:0

Reply:
I think that filtering by `__typename: 'WorkerStatusActive'` will be more future-proof. The `phantom` was used to overcome Hydra schema shortcomings AFAIR.

Reply:
https://dzhelezov.gitbook.io/hydra/docs/schema-spec/variant-types
I was not able to filter by __typename, but maybe will be able to intoduce isTypeOf after you changes for nested filtering